test/tap/lifecycle.js

```diff
@@ -42,3 +42,30 @@ test('lifecycle : rejects wd for ', function (t) {
     t.end()
   })
 })
+
+test('lifecycle : accepts TMPDIR that is writable', function (t) {
```

Comment:
These tests assume TMPDIR is set properly and writeable, hopefully that's not a problem anywhere.